onnx/checker.py

```diff
@@ -80,36 +82,39 @@ def check_tensor(tensor: TensorProto, ctx: C.CheckerContext = DEFAULT_CONTEXT) -
 
 
 def check_attribute(
-    attr: AttributeProto, ctx: C.CheckerContext = DEFAULT_CONTEXT
+    attr: AttributeProto,
+    ctx: C.CheckerContext = DEFAULT_CONTEXT,
+    lex_ctx: C.LexicalScopeContext = LEXICAL_SCOPE_CONTEXT,
```

Comment:
Sorry for being late to point this out but we should avoid abbreviations for readability. Consider `lexical_scope_ctx` (keep `ctx` for consistency)?

If @gramalingam likes the suggestion, I suggest creating a follow up PR to amend this change.

Reply:
`lexical_scope_ctx` sounds good to me. I could work on this. Same with parameterizing the checker tests